### PR TITLE
Added lecture name to the header of view page

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr adds the lecture name to the header of the view page.

### CODE REVIEW

File: src/main/webapp/view/view.js [hunk 1]
```diff
@@ -53,3 +54,9 @@ function getLectureId() {
   const urlParams = new URLSearchParams(window.location.search);
   return urlParams.get(PARAM_ID);
 }
+
+/** Sets the lecture name in `header-text`. */
+function setLectureName() {
+  const headerText = document.getElementById('header-text');
+  headerText.innerText = window.LECTURE.lectureName;
```
Comment (n-wach) [inline src/main/webapp/view/view.js:61]: Can you remove the `href` attribute so clicking doesn't take us to the About page?

Comment (n-wach) [reply src/main/webapp/view/view.js:61]: https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute

Comment (jaysaleh) [reply src/main/webapp/view/view.js:61]: It is just a `<h2>` right now

Comment (n-wach) [reply src/main/webapp/view/view.js:61]: Ah yes, approved.

**Comment by n-wach** [comment]
Oops, sorry Jay I accidentally merged my branch before this was in master.  Should still be fine tho, #217 was approved.

**Pull request merged**